test-data/unit/check-serialize.test

```diff
@@ -1035,9 +1035,9 @@ D = TypedDict('D', {'x': int, 'y': str}, total=False)
 d: D
 [builtins fixtures/dict.pyi]
 [out1]
-main:2: error: Revealed type is 'TypedDict(x=builtins.int, y=builtins.str, _fallback=m.D, _total=False)'
+main:2: error: Revealed type is 'TypedDict('m.D', {'x'?: builtins.int, 'y'?: builtins.str})'
```

Comment:
I think while the `?`-syntax for non-required keys has its downsides, it works well in this case.
We need to be concise and I think `?` is pretty intuitive. 
It might be worth mentioning it documentation.